autoload/syncopate.vim

```diff
@@ -89,6 +89,13 @@ function! s:InformUserAboutCopiedText(first, last)
   endif
 endfunction
 
+" wait to remove file
+function! s:WaitToRemoveFile(file)
+  sleep 1000m
```

Comment:
Adding a new plugin variable sounds good.  You could also make it a floating point variable -- say, `open_browser_delay_seconds` -- and default it to `0.0`.  Be sure to add documentation in `flags.vim` in the same way that the others are documented.  It would be great if you could also re-run `vimdoc` to update the online help.